core-api/src/test/java/com/optimizely/ab/config/audience/AudienceConditionEvaluationTest.java

```diff
@@ -228,6 +244,21 @@ public void gtMatchConditionEvaluatesTrue() throws Exception {
         assertNull(testInstanceInteger.evaluate(badAttributes));
     }
 
+    /**
+     * Verify that UserAttribute.evaluate for GT match type returns null if the UserAttribute's
+     * value type is invalid number.
+     */
+    @Test
+    public void gtMatchConditionEvaluatesNullWithInvalidAttr() throws Exception {
+        BigInteger bigInteger = new BigInteger("33221312312312312");
+        Double infiniteDouble = 1.2/0;
```

Comment:
Technically this evaluates to `NaN`, not infinity. It's probably cleaner use `Double.NaN`, `Double.POSITIVE_INFINITY`, and `Double.NEGATIVE_INFINITY` where needed